### ISSUE TITLE
Logging doesn't indicate profile vs policy

### ISSUE THREAD

**Issue by caseydavenport** [projectcalico/felix#1324]
```
calico.felix.profilerules 270: Updating chains for profile default/np.projectcalico.org/cali-ns-2.allow-client5-cali-ns-2-prometheus
```

Not super urgent, but a bit confusing that Policy objects created by the user end up being logged out as "profile".

If we can't distinguish between them at that point in the code, perhaps we should just not say the word "profile", or say "profile or policy" or something like that.

**Comment by fasaxc** [comment]
That code is already gone from master so I think we should live with, WDYT?

**Comment by caseydavenport** [comment]
Yep, so long as this is "fixed" in v2.1.0, I'm happy :)
